lib/compat/wordpress-6.5/scripts-modules.php

```diff
@@ -0,0 +1,304 @@
+ *                                                                                 this module. The dependencies can
+ *                                                                                 be strings or arrays. If they are
+ *                                                                                 arrays, they need an `id` key with
+ *                                                                                 the module identifier, and can
+ *                                                                                 contain an `import` key with either
+ *                                                                                 `static` or `dynamic`. By default,
+ *                                                                                 dependencies that don't contain an
+ *                                                                                 `import` key are considered static.
+ * @param string|false|null                                             $version   Optional. String specifying the
+ *                                                                                 module version number. Defaults to
+ *                                                                                 false. It is added to the URL as a
+ *                                                                                 query string for cache busting
+ *                                                                                 purposes. If $version is set to
+ *                                                                                 false, the version number is the
+ *                                                                                 currently installed WordPress
+ *                                                                                 version. If $version is set to
+ *                                                                                 null, no version is added.
+ */
+function gutenberg_enqueue_module( $module_id, $src = '', $deps = array(), $version = false ) {
+	gutenberg_modules()->enqueue( $module_id, $src, $deps, $version );
+}
+
+/**
+ * Unmarks the module so it is no longer enqueued in the page.
+ *
+ * @since 6.5.0
+ *
+ * @param string $module_id The identifier of the module.
+ */
+function gutenberg_dequeue_module( $module_id ) {
+	gutenberg_modules()->dequeue( $module_id );
+}
+
+/**
+ * Add module fields from block metadata to WP_Block_Type settings.
+ *
+ * This filter allows us to register modules from block metadata and attach additional fields to
+ * WP_Block_Type instances.
+ *
+ * @param array $settings Array of determined settings for registering a block type.
+ * @param array $metadata Metadata provided for registering a block type.
+ */
+function gutenberg_filter_block_type_metadata_settings_register_modules( $settings, $metadata = null ) {
```

Comment:
Same here. This is not in WP 6.5, so maybe it should not be here.

Reply:
Moved back to experimental.